cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -20,6 +20,77 @@
 #include "cartographer_ros/msg_conversion.h"
 #include "cartographer_ros/time_conversion.h"
 
+#include <pcl/PCLPointCloud2.h>
+#include <pcl/conversions.h>
+
+namespace pcl_conversions {
+  inline
+  void toPCL(const builtin_interfaces::msg::Time &stamp, pcl::uint64_t &pcl_stamp)
+  {
+    pcl_stamp = stamp.sec * 1000000 + stamp.nanosec / 1000ull;
+  }
+
+  inline
+  void toPCL(const sensor_msgs::msg::PointField &pf, pcl::PCLPointField &pcl_pf)
+  {
+    pcl_pf.name = pf.name;
+    pcl_pf.offset = pf.offset;
+    pcl_pf.datatype = pf.datatype;
+    pcl_pf.count = pf.count;
+  }
+
+  inline
+  void toPCL(const std::vector<sensor_msgs::msg::PointField> &pfs, std::vector<pcl::PCLPointField> &pcl_pfs)
+  {
+    pcl_pfs.resize(pfs.size());
+    std::vector<sensor_msgs::msg::PointField>::const_iterator it = pfs.begin();
+    int i = 0;
+    for(; it != pfs.end(); ++it, ++i) {
+      toPCL(*(it), pcl_pfs[i]);
+    }
+  }
+
+  inline
+  void toPCL(const std_msgs::msg::Header &header, pcl::PCLHeader &pcl_header)
+  {
+    toPCL(header.stamp, pcl_header.stamp);
+    // FIXME: Seq doesn't exist anymore
```

Comment:
Please consider putting your github username in parenthesis like this: `// FIXME(clalancette): ...`. The idea is that we can easily know who to ask about it if the comment is no longer clear, without resorting to using `git blame`. If you don't see a need for it then that's totally fine, but I personally think it's a good habit to get into.